pkg/minikube/problem/err_map.go

```diff
@@ -109,6 +109,12 @@ var vmProblems = map[string]match{
 		Issues: []int{4511},
 		GOOS:   []string{"windows"},
 	},
+  
+	"HYPERV_FILE_ACCESS_DENIED": {
+		Regexp: re(`The process cannot access the file because it is being used by another process`),
+		Advice: "Hyper-V Manager locked the file. Please stop the Hyper-V Manager, and run 'minikube delete'.",
+		URL:    "https://docs.docker.com/machine/drivers/hyper-v/",
+    
```

Comment:
`[/home/travis/gopath/src/github.com/kubernetes/minikube/pkg/minikube/problem/err_map.go:323:2: missing ',' before newline in composite literal /home/travis/gopath/src/github.com/kubernetes/minikube/pkg/minikube/problem/err_map.go:326:1: expected `

This is missing a line that says:

```},```